test/integration/change-streams/change_stream.test.ts

```diff
@@ -1816,6 +1829,78 @@ describe('Change Streams', function () {
       });
     });
   });
+
+  describe("NODE-4763 - doesn't produce duplicates after resume", function () {
+    const resumableError = { code: 6, message: 'host unreachable' };
+
+    beforeEach(async function () {
+      await client.db('admin').command({
+        configureFailPoint: is4_2Server(this.configuration.version)
+          ? 'failCommand'
+          : 'failGetMoreAfterCursorCheckout',
+        mode: { skip: 1 },
```

Comment:
Why is `skip: 1` necessary for these tests?

Reply:
This way, we can make the second getMore call fail, i.e., we control the resumption process. These two snippets are functionally equivalent:
```javascript
// 1.
const change1 = await changeStream.next();
await setFailPoint({ times: 1 });
const change2 = await changeStream.next(); // fails and resumes

// 2.
await setFailPoint({ skip: 1 });
const change1 = await changeStream.next();
const change2 = await changeStream.next(); // fails and resumes
await setFailPoint("off");
```

While the first is more explicit and easier to follow, the second is arguably more readable. Let me know if you prefer option 1; I’m fine with both and don’t have a strong preference.

Reply:
No preference from me.  If we choose to keep it, can we add a comment explaining why we chose to add it?